Sources/Core/SignedNumeric.swift

```diff
@@ -0,0 +1,14 @@
+// Copyright © 2022 Max Desiatov <max@desiatov.com>.
+// All Rights Reserved.
+// SPDX-License-Identifier: BSD-3
+
+public protocol SignedNumeric: Numeric {
+  static prefix func - (_ operand: Self) -> Self
+
+  mutating func negate()
+}
+
+// FIXME: extension with default implementations of `prefix func -` and
```

Comment:
I tend to prefer to use `FIXME(author)` or `FIXME(issue)` so that we have a point of reference from this to something more detailed for someone wanting to pick it up.  In the case that there is no associated issue filed, then the author can serve as a starting point.\

```suggestion
// FIXME(MaxDesiatov): extension with default implementations of `prefix func -` and
```

This is where the suggested extensions for VSCode comes in :).  It will extract all the FIXMEs so that someone can easily find them across the project.